src/main/java/anab/Person/ManagePerson.java

```diff
@@ -2,8 +2,23 @@
 
 public class ManagePerson {
 
+    static void printeazaText(String name) {
+        System.out.println("Hello!" + name);
+    }
+
     public static void main(String[] args) {
-        Person person = new Person();
+        Person personPerson = new Person("Ion", "Popescu", "Gender", 15, "EmailAddress");// create an object of Person class
+
+        System.out.println("First Name:\t" + personPerson.getFirstName());
+        System.out.println("Last Name:\t" + personPerson.getLastName());
+        String name = "Buru";
+        printeazaText("Ana" + name);
```

Comment:
output will be "AnaBuru"; can add a space: "Ana "

Reply:
Space added and changed displayed format.